common/scripts/setup_env.sh

```diff
@@ -26,13 +26,13 @@ set -e
 LOCAL_ARCH=$(uname -m)
```

Comment:
Once this is passing CI properly, please submit the PR to https://github.com/istio/common-files. The PR will then be replicated to all of our repos and automatically merged after approval.

Reply:
Will do!

Reply:
https://github.com/istio/common-files/pull/228